rali/rali_pybind/example/tf_mnistTrainingExample/mnist_tfrecord.py

```diff
@@ -0,0 +1,119 @@
+#! /usr/env/bin python3
+
+"""Convert MNIST Dataset to local TFRecords"""
+
+import argparse
+import os
+import sys
+import tensorflow as tf
+from tensorflow.examples.tutorials.mnist import input_data
+
+def _data_path(data_directory:str, name:str) -> str:
+    """Construct a full path to a TFRecord file to be stored in the 
+    data_directory. Will also ensure the data directory exists
+    
```

Comment:
Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6366586)

Reply:
Please ignore this. It is complaining about commented block